packages/services/api/src/modules/audit-logs/module.graphql.mappers.ts

```diff
@@ -0,0 +1,3 @@
+import { AuditLogModel } from './providers/audit-logs-manager';
+
+export type AuditLogMapper = AuditLogModel;
```

Comment:
When using Server Preset, it can automatically match up the GraphQL type with the mapper type using this convention: `<GraphQL type>Mapper`

This allows us to declare all our mappers in the code and avoid adding them to the config (which doesn't have import check or type-safety, etc.). In our case, it'd look like this:

```
export type AuditLogMapper = AuditLogModel;
export type UserInvitedAuditLogMapper = AuditLogModel;
export type UserJoinedAuditLogMapper = AuditLogModel;
export type UserRemovedAuditLogMapper = AuditLogModel;
// ... and other similar mappers
```